litex_boards/platforms/digilent_zybo_z7.py

```diff
@@ -113,12 +157,22 @@ class Platform(Xilinx7SeriesPlatform):
     def __init__(self, variant="z7-20", toolchain="vivado"):
         device = {
             "z7-10": "xc7z010-clg400-1",
-            "z7-20": "xc7z020-clg400-1"
+            "z7-20": "xc7z020-clg400-1",
+            "original": "xc7z010-clg400-1"
         }[variant]
+        ps7_config = ps7_config_variants["common"]
+        if variant == "original":
+            _connectors = _connectors + _connectors_original
+            _io = _io + _io_original
+            ps7_config.update(ps7_config_variants["original"])
+        else:
+            _connectors = _connectors + _connectors_z7
+            _io = _io + _io_z7
+            ps7_config.update(ps7_config_variants["z7"])
         Xilinx7SeriesPlatform.__init__(self, device, _io,  _connectors, toolchain=toolchain)
         self.add_extension(_ps7_io)
         self.add_extension(_usb_uart_pmod_io)
-        self.ps7_config = ps7_config[variant]
+        self.ps7_config = ps7_config
```

Comment:
Before a small warning: `_ps7_io` isn't declared before.

Maybe to have something more compact / clear:
```
ps7_config.update(ps7_config_variants["original" if variant == original else "z7"])

Xilinx7SeriesPlatform.__init__(self, device, _io, _connectors, toolchain=toolchain)
self.add_extension(_io_original if variant == "original" else _io_z7)
self.add_connector(_connector_original if variant == "original" else _connector_z7)
self.add_extension(_usb_uart_pmod_io)
self.ps7_config = ps7_config
```